### ISSUE TITLE
Put string loading in common

### ISSUE THREAD

**Issue by npresseault** [mirego/trikot.kword#25]
At this moment, we have a swift extension for iOS, AndroidKWord for android and JSWord for javascript. All 3 have overlaps and could be put in common.

**Comment by Boubalou** [comment]
Good point! A simple actual/expect regarding the loading of the translation file should be enough to keep only one implementation in commonMain! 💯
